src/functions-and-arrays.js

```diff
@@ -1,19 +1,90 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(arg1 , arg2) {
+  if (arg1 > arg2) {
+    return arg1
+  } else if (arg1 < arg2){
+    return arg2
+  } else return arg1
+
+}
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+let longestWord = 0
+function findLongestWord( words ) {
+  if (words.length === 0){
+    return null 
+  } else if (words.length === 1){
+  return words[0]
+  } else {
+    for (var i=0; i < words.length; i++){
+      if (longestWord < words[i].length){ 
+        longestWord=words[i].length
+      } 
+        return longestWord
```

Comment:
Error de nomenclatura, no de código. Piensas que estás retornando una palabra pero estás retornando un número.

Tendrías que tener un console.log(longestWord) en la línea 24, para ver que retornas en realidad.
Jasmine tmb te lo dice: 
![image](https://user-images.githubusercontent.com/26923217/104712219-95489a80-5722-11eb-8c5a-6ac4182fe59e.png)
Esperaba "foo" y se ha encontrado un 3.

Esto es lo que estás haciendo. (fijate que he metido la variable dentro de la función, NUNCA las declares fuera a no ser que sea indispensable)
![image](https://user-images.githubusercontent.com/26923217/104712372-c032ee80-5722-11eb-8968-e6e5768afb98.png)

Es mejor guardarte la palabra y comparar la length, Que guardarte solo la length, Pierdes información.
![image](https://user-images.githubusercontent.com/26923217/104712691-261f7600-5723-11eb-9236-1779fee14fcc.png)

Y sacamos el return del for. Para retornar el resultado final!